reference/forms/types/entity.rst

```diff
@@ -202,9 +202,10 @@ query_builder
 Allows you to create a custom query for your choices. See
 :ref:`ref-form-entity-query-builder` for an example.
 
-The value of this option can either be a ``QueryBuilder`` object, a Closure or null.
+The value of this option can either be a ``QueryBuilder`` object, a Closure or ``null``.
 When using a Closure, you will be passed the ``EntityRepository`` of the entity
-as the only argument and should return a ``QueryBuilder``.  If you'd like to display a list of empty entries, you can return null in the query_builder closure.
+as the only argument and should return a ``QueryBuilder``.  
+If you'd like to display a list of empty entries, you can return ``null`` in the closure.
```

Comment:
When you set the option to `null`, but not when return `null` in the closure, isn't it?